tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "target": "es6",
         "outDir": "out",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
Graph client uses some DOM type references underneath the hood... this solved the related compile errors I was getting

Reply:
Oh that's unfortunate. I wish there was a way to only include the exact part of the DOM lib we actually need.

Reply:
I'd at least leave a comment in the tsconfig about it, but this feels bad.